test/gtest/ucm/malloc_hook.cc

```diff
@@ -348,6 +352,8 @@ UCS_TEST_F(malloc_hook, multi_threads) {
     ucs::ptr_vector<test_thread> threads;
     pthread_barrier_t barrier;
 
+    skip_on_bistro();
```

Comment:
there is not enough test coverage of bistro
need to update single/multi thread tests to call all patched glibc funcs instead of malloc, in case of bistro (it already calls shmdt directly for example)

Reply:
added multi-threaded tests